backend/routers/projectRouter.js

```diff
@@ -39,10 +41,10 @@ router.get('/', async (req, res) => {
 */
 router.post('/', async (req, res) => {
   try {
-    const { title } = req.body
+    const { title, apiKey } = req.body
 
     if (!title) return res.status(400).json({ msg: 'Missing project title.' })
```

Comment:
I've just realised that we dont have project titles. Would it be possible to remove this validation?